.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [
+      1,
+      {
+        "js": "always",
+        "json": "always"
+      }
+    ]
+  },
+  "ignorePatterns": ["dist/", "build/"]
+}
```

Comment:
While you did add this file, ESLint is still not part of your `package.json` list of dependencies. You need to follow [these instructions](https://github.com/microverseinc/linters-config/blob/master/html-css-js/README.md), step-by-step, from the beginning, in order to properly set up the linters for this project. That means you'll need to replace both the `linters.yml` file and update the `package.json` file. Please do so.